spec/system/provider_interface/provider_removes_conditions_from_an_offer_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'rails_helper'
+
+RSpec.feature 'Remove offer conditions' do
+  include CourseOptionHelpers
+  include DfESignInHelpers
+  include ProviderUserPermissionsHelper
+
+  scenario 'Provider user removes the conditions from an offer', js: true do
```

Comment:
can we amend the scenario description for both those specs to differentiate between js on/off, same for spec below as now the scenarios are described exactly the same

Reply:
yes!